packages/page-pattern-modal/src/styles/page-pattern-modal.scss

```diff
@@ -35,12 +35,24 @@ $sidebar-width-desktop: 324px;
 
 // Overrides of the Gutenberg modal component
 .page-pattern-modal {
+	position: relative;
+
+	&.components-modal__frame {
+		max-height: none;
+	}
+
+	@media screen and ( max-width: $breakpoint-mobile ) {
+		height: calc( 100vh - 55px );
+		top: 50px; // don't get hidden by the adminbar
+	}
+
 	@media screen and ( min-width: $breakpoint-mobile ) {
 		width: calc( 100% - 32px );
 		height: 90vh;
 		// prevents a 1px strip of content being visible after scrolling
 		overflow-y: hidden;
 		padding-bottom: 24px;
+		top: 20px;
```

Comment:
I am missing what this does?  Removing this part I dont see any difference, but I could easily be missing something. 😆

Reply:
This and some other confusing stuff was to compensate for the adminbar, which I was seeing because I was going straight to `wp-admin`, but it can be removed now too

Reply:
I think it just nicely aligns the top of the modal with the editor toolbar in the background, maybe?

<img width="300" alt="image" src="https://user-images.githubusercontent.com/1699996/122291970-0f8c6f00-cebb-11eb-8bcc-e2383861c7a4.png">